src/server/api/util/airtable.js

```diff
@@ -68,6 +72,42 @@ async function fetchAirtableCMSItems(viewName, { pageSize = 100, offset }) {
   };
 }
 
+async function fetchAirtableCMSTags({ pageSize = 100, offset }) {
+  const results = await axios.get(
+    `https://api.airtable.com/v0/${AIRTABLE_CMS_BASE_ID}/${AIRTABLE_CMS_TAGS_TABLE_ID}`,
+    {
+      headers: {
+        Authorization: `Bearer ${AIRTABLE_API_SECRET}`,
+      },
+      params: {
+        pageSize,
+        view: 'All',
+        offset,
+      },
+    }
+  );
+
+  const normalizedRecords = results.data.records.map(({ fields }) => {
+    const id = fields.ID;
+    const name = fields.Name;
+    const nameEn = fields['Name (Eng)'];
+    const isPublic = fields.Public;
```

Comment:
Should `isPublic` be filtered here instead of frontend?

Reply:
`isPublic` means the tag is unlisted, the tag is not showing in the navigation, but can be accessed through the URL